resources/assets/coffee/osu_common.coffee

```diff
@@ -170,8 +170,8 @@
     el.outerHTML
 
 
-  linkify: (text) ->
-    return text.replace(osu.urlRegex, '<a href="$1" rel="nofollow">$1</a>')
+  linkify: (text, newWindow = false) ->
+    return text.replace(osu.urlRegex, "<a href=\"$1\" rel=\"nofollow\"#{if newWindow then ' target=\"_blank\"' else ''}>$2</a>")
```

Comment:
Should add a `noopener` rel. Also maybe replace the double quotes inside with single quotes (and the escape inside target=_blank isn't needed).

`return` keyword optional.